mpas_analysis/analysis_task_template.py

```diff
@@ -0,0 +1,133 @@
+'''
+This is an example analysis task to be used as a template for new tasks.
+It should be copied into one of the component folders (`ocean`, `sea_ice`,
+`land_ice`, etc.) and modified as needed.
+
+Don't forget to remove this docstring. (It's not needed.)
+
+Authors
+-------
+Xylar Asay-Davis
+'''
+
+# import python modules here
+
+# import mpas_analysis module here (those with relative paths starting with
+# dots)
+from ..shared.analysis_task import AnalysisTask
+
+
```

Comment:
I disagree with the change from `MyTask` to `MyTaskName` because the ending `Name` is appropriate for a string containing the task name but not for a class that implements a task.  So the class really should be thought of as *being* the task (as opposed to just having the same name as the task).